### ISSUE TITLE
Add Batch controls Back into my files listing

### ISSUE THREAD

**Issue by cam156** [projecthydra/sufia#1097]
Batch controls were out of scope for #999 so they were changed to TODOs.

Add the batch controls back into the dashboard.

The Add To Collection flow should still work for works, so test that flow.  If it does not work, create and additional ticket for making that work.

related ticket: #1096, #1003

Depends on #1093

**Comment by cam156** [comment]
related to #1104
